portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,86 @@
+html {
+    background: url(images/try.gif);
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+
+
+h1 {
+    font-size: 100px;
+    color: white;
+    padding-top: 12%
+}
+
+h2 {
+    font-size: 50px;
```

Comment:
the nittiest of nits: alphabetizing declarations https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order